bindata/v3.11.0/kube-apiserver/pod.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       httpGet:
         scheme: HTTPS
         port: 6443
-        path: healthz/ready
+        path: healthz
```

Comment:
pull this commit into a another PR and merge it.

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/pull/118